staging/src/k8s.io/kubectl/pkg/cmd/alpha.go

```diff
@@ -36,7 +35,7 @@ func NewCmdAlpha(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 
 	// Alpha commands should be added here. As features graduate from alpha they should move
 	// from here to the CommandGroups defined by NewKubeletCommand() in cmd.go.
-	cmd.AddCommand(debug.NewCmdDebug(f, streams))
+	//cmd.AddCommand(example.NewCmdExample(f, streams))
```

Comment:
I wonder if we can make this work for both places at least for one release. I know it's not a hard requirement, but I think it's good UX to have both working. The alpha one can have a deprecation warning.